### PR TITLE
fix: replace require with import for blue_modules

### PR DESCRIPTION
**Pull request by limpbrains**
Why ? Because require doesn't work in Typescript

Also converted a couple of small modules to Typescript.

### CODE REVIEW

File: blue_modules/analytics.ts [hunk 1]
```diff
@@ -20,7 +20,6 @@ if (process.env.NODE_ENV !== 'development') {
     }
 
     Bugsnag.start({
-      collectUserIp: false,
```
Comment (limpbrains) [inline blue_modules/analytics.ts:23]: Browser only

File: screen/lnd/ldkInfo.tsx [hunk 2]
```diff
@@ -314,9 +317,11 @@ const LdkInfo = () => {
       params: {
         availableWallets,
         chainType: Chain.ONCHAIN,
-        onWalletSelect: (selectedWallet: AbstractWallet) => {
+        onWalletSelect: (selectedWallet: TWallet) => {
           const selectedWalletID = selectedWallet.getID();
-          selectedWallet.getAddressAsync().then(selectWallet.setRefundAddress);
+          selectedWallet.getAddressAsync().then(address => {
+            wallet.setRefundAddress(address as string);
+          });
```
Comment (limpbrains) [inline screen/lnd/ldkInfo.tsx:324]: Did I fix it right?

File: screen/wallets/addMultisigStep2.js [hunk 3]
```diff
@@ -468,7 +469,7 @@ const WalletsAddMultisigStep2 = () => {
 
   const scanOrOpenFile = () => {
     if (isDesktop) {
-      fs.showActionSheet({ anchor: findNodeHandle(openScannerButton.current) }).then(onBarScanned);
```
Comment (limpbrains) [inline screen/wallets/addMultisigStep2.js:471]: Am I fixing it correctly ? @marcosrdz 
`fs.showActionSheet` was removed here #5327

File: screen/send/psbtMultisigQRCode.js [hunk 4]
```diff
@@ -63,7 +63,7 @@ const PsbtMultisigQRCode = () => {
 
   const openScanner = () => {
     if (isDesktop) {
-      fs.showActionSheet({ anchor: findNodeHandle(openScannerButton.current) }).then(data => onBarScanned({ data }));
```
Comment (limpbrains) [inline screen/send/psbtMultisigQRCode.js:66]: @marcosrdz same

**Comment by limpbrains** [comment]
What does this e2e build error means ?
```
Failed to load signer "signer #1"
java.io.FileNotFoundException: detox.keystore (No such file or directory)
	at java.base/java.io.FileInputStream.open0(Native Method)
	at java.base/java.io.FileInputStream.open(FileInputStream.java:219)
	at java.base/java.io.FileInputStream.<init>(FileInputStream.java:157)
	at java.base/java.io.FileInputStream.<init>(FileInputStream.java:112)
	at com.android.apksigner.SignerParams.loadKeyStoreFromFile(SignerParams.java:382)
	at com.android.apksigner.SignerParams.loadPrivateKeyAndCertsFromKeyStore(SignerParams.java:269)
	at com.android.apksigner.SignerParams.loadPrivateKeyAndCerts(SignerParams.java:202)
	at com.android.apksigner.ApkSignerTool.getSignerConfig(ApkSignerTool.java:438)
	at com.android.apksigner.ApkSignerTool.sign(ApkSignerTool.java:353)
	at com.android.apksigner.ApkSignerTool.main(ApkSignerTool.java:92)
Failed to load signer "signer #1"
java.io.FileNotFoundException: detox.keystore (No such file or directory)
	at java.base/java.io.FileInputStream.open0(Native Method)
	at java.base/java.io.FileInputStream.open(FileInputStream.java:219)
	at java.base/java.io.FileInputStream.<init>(FileInputStream.java:157)
	at java.base/java.io.FileInputStream.<init>(FileInputStream.java:112)
	at com.android.apksigner.SignerParams.loadKeyStoreFromFile(SignerParams.java:382)
	at com.android.apksigner.SignerParams.loadPrivateKeyAndCertsFromKeyStore(SignerParams.java:269)
	at com.android.apksigner.SignerParams.loadPrivateKeyAndCerts(SignerParams.java:202)
	at com.android.apksigner.ApkSignerTool.getSignerConfig(ApkSignerTool.java:438)
	at com.android.apksigner.ApkSignerTool.sign(ApkSignerTool.java:353)
	at com.android.apksigner.ApkSignerTool.main(ApkSignerTool.java:92)
```

File: screen/wallets/addMultisigStep2.js (hunk 3 above)
Comment (marcosrdz) [reply screen/wallets/addMultisigStep2.js:471]: Need to pass options. look at sendButtonLongPress on list.js

Comment (limpbrains) [reply screen/wallets/addMultisigStep2.js]: fixed

File: screen/send/psbtMultisigQRCode.js (hunk 4 above)
Comment (limpbrains) [reply screen/send/psbtMultisigQRCode.js]: fixed

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/atqo5mkk24vqvpagzhj6ldi3sm?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/1035)

**Review by marcosrdz — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Comment by Overtorment** [comment]
Nice! I wonder if we get any performance gains as well with this

**Pull request merged**